frontend/src/views/NotificacaoForm.vue

```diff
@@ -5,6 +5,18 @@
     </h3>
     <base-page>
       <v-form ref="form">
+        <identificacao-notificante
+          :notificacao="notificacao"
+          @update:unidadeSaudeId="updateUnidadeSaude('unidadeSaudeId', $event)"
```

Comment:
Isso aqui não vai funcionar, vai sempre setar o valor fixo ''unidadeSaudeId" na variável.
Altera pra
```suggestion
          @update:unidadeSaudeId="updateUnidadeSaude"
```